lib/grizzly/zwave/commands/switch_multilevel_report.ex

```diff
@@ -85,6 +85,15 @@ defmodule Grizzly.ZWave.Commands.SwitchMultilevelReport do
     end
   end
 
+  # no spec
+  def decode_params(<<_value_byte, _target_value_byte, duration::binary>>) do
+    Logger.warning(
+      "[Grizzly] Unexpected trailing bytes in SwitchMultilevelReport: #{inspect(duration)}"
+    )
+
+    {:error, %DecodeError{value: duration, param: :duration, command: :switch_multilevel_report}}
```

Comment:
I think we still want to handle the frame if there are trailing bytes instead of erroring out (based on the assumption that the value and target value params are accurate). If the device in question always sends the extra byte, then we'd never be able to receive a switch multilevel report (even a malformed one) from it, which I don't think is what we want.

What I was imagining was given `<<0x32, 0x63, 0x0, 0x87>>`, we would parse that as value=50, target_value=99, and duration=0. Sorry for the misunderstanding there.

Reply:
> If the device in question always sends the extra byte, 
Good point